app/build.gradle.kts

```diff
@@ -54,13 +54,13 @@ android {
     javaMaxHeapSize = "4g"
   }
 
-  if (buildFromSource.toBoolean()) {
     packagingOptions {
-      pickFirst("**/libc++_shared.so")
-      pickFirst("**/libmapbox-common.so")
+      if (buildFromSource.toBoolean()) {
+        jniLibs.pickFirsts.add("**/libc++_shared.so")
+      }
+      jniLibs.pickFirsts.add("**/libmapbox-common.so")
```

Comment:
we have similar line in `sdk` module - do we actually need it here?

Reply:
Yes, the config in sdk module is for Android test